Add missing prototypes so compile doesn't complain.

Patch by Kevin McCarthy <signals@gentoo.org>

--- src/ui-gtk.h
+++ src/ui-gtk.h
@@ -131,4 +131,7 @@
 
 int key_handle_key_press(GtkWidget *wid, GdkEventKey *evt, window_t *sess);
 
+void mg_changui_new(window_t *sess, int tab, int focus);
+void fe_close_window(window_t *sess);
+
 #endif
